apps/src/levelbuilder/level-editor/BubbleChoiceDescriptionEditor.jsx

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+import React, {useEffect, useRef} from 'react';
+
+import initializeCodeMirror from '@cdo/apps/code-studio/initializeCodeMirror';
+
+import styles from './edit-child-level-settings.module.scss';
+
+const BubbleChoiceDescriptionEditor = ({
+  description,
+  index,
+  handleDescriptionChange,
+}) => {
+  const isInitialized = useRef(false);
+
+  useEffect(() => {
+    if (isInitialized.current) return;
+
+    const elementId = `bubble_choice_description_${index}`;
+    const element = document.getElementById(elementId);
+    if (element) {
+      initializeCodeMirror(elementId, 'markdown', {
+        callback: codeMirror => {
+          handleDescriptionChange(index, codeMirror.getValue());
+        },
+      });
+      isInitialized.current = true;
+    } else {
+      console.error(`Element with ID ${elementId} not found.`);
```

Comment:
Not important, not worth changing, but raising an exception here might be sliiiiiightly better than a console.error message. In general, its better to propagate exceptions vs silently printing to the user's local error console. During dev we use console.error() a lot more than we might after merging to prod.